cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java

```diff
@@ -46,6 +46,7 @@
     public static final Transformation DELETE_TRANSFORMATION = new Transformation().width(100).crop("scale").overlay(new TextLayer().text(SUFFIX + "_delete").fontFamily("Arial").fontSize(60));
     public static final String TEST_KEY = "test-key" + SUFFIX;
     public static final String API_TEST_RESTORE = "api_test_restore" + SUFFIX;
+    public static final Set<String> createdFolders = new HashSet<String>();
```

Comment:
@michalkcloudinay , you only create a single folder with a known name in your test("api_test_create_folder" + "_" + SUFFIX), why you need to store a Set?

Reply:
@asisayag2 In order to support additional functions (in the future) where we create folders. All folders will be added to this set and will be deleted in one command when cleaning up.

Reply:
@michalkcloudinay , not needed right now. 
Lean approach... we add what we need, and the next test that creates a folder will refactor to handle >1 folders. 
@nitzanj , WDYT?